Before: 95fe5a83ed2969f958bcc2ee06166ee16f00c6552c33a7b88d37d2003af66318
After: f57076598c46742495f30a0e6bb0a6806a961a613304fcb3863b2ea48688d4e6
Patch: 2621891d46c79f9186f080d74ebf1fe4a7a79784c6fe375d79e044a94cb7e9cf
--- test/bootstrap/all.rb	2012-07-22 20:44:49.268917458 +1000
+++ test/bootstrap/all.rb	2012-07-22 21:28:25.320041400 +1000
@@ -1,11 +1,6 @@
 # encoding: utf-8
 APP_ROOT = File.expand_path('../../..',__FILE__)
 require 'rubygems'
-require 'bundler/setup'
 require 'corvid/test/bootstrap/all'
 
 Bundler.require :default
-
-# Load test helpers
-Dir.glob("#{APP_ROOT}/test/helpers/**/*.rb") {|f| require f}
-
